### PR TITLE
Implementing Unit Testing Version 2 final

### PR DESCRIPTION
**Pull request by ghost**
Issue:
	Title: Unit Test Implementation using mock
	Link: https://github.com/XgridInc/xc3/issues/4

	[PS-1]
	- Added unit tests for the following services:
		- Loadbalancer
		- API Gateway

<br>

- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Every function, interface, class has a comment describing what it does and input/output parameters

Signed-off-by: Kashan Ali [ kashan.ali@xgrid.co ]

### CODE REVIEW

File: tests/unit_tests/config.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import json
```
Comment (samanXgrid) [inline tests/unit_tests/config.py:1]: please add license

File: tests/unit_tests/unit-test-api-gatway.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import logging
```
Comment (samanXgrid) [inline tests/unit_tests/unit-test-api-gatway.py:1]: please add license

File: tests/unit_tests/unit-test-loadbalancer.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import logging
+import unittest
```
Comment (samanXgrid) [inline tests/unit_tests/unit-test-loadbalancer.py:2]: please add license

Comment (kashan-01) [reply tests/unit_tests/unit-test-loadbalancer.py:2]: sure !

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by samanXgrid — APPROVED** [review]
LGTM

**Review by mfaaraz — APPROVED** [review]
LGTM

**Review by mfaaraz — APPROVED** [review]
LGTM

**Pull request merged**